openapi.yml

```diff
@@ -690,3 +736,19 @@ components:
           $ref: '#/components/schemas/Interface'
         endpoint_b:
           $ref: '#/components/schemas/Interface'
+    Tag_ranges: # Can be referenced via '#/components/schemas/Tag_ranges'
+      type: object
+      required:
+        - tag_type
+        - tag_ranges
+      properties:
+        tag_type:
+          type: string
+        tag_ranges:
+          type: array
+          items:
+            anyOf:
+              - type: array
+              - type: integer
+          example: [[1, 500], 2096, [3001]]
```

Comment:
Cool. It's great how this type spec ended up for `tag_ranges` allowing anyof array or integer. It's very readable and high-level while still being easy to transform for the format the back-end expects

Reply:
Added restriction for the length not to be empty.